schemas/com.snowplowanalytics.iglu/client_error/jsonschema/1-0-0

```diff
@@ -0,0 +1,101 @@
+{
+	"$schema": "http://iglucentral.com/schemas/com.snowplowanalytics.self-desc/schema/jsonschema/1-0-0#",
+	"description": "Possible unsuccessful result produced by Iglu Client",
+    "self": {
+        "vendor": "com.snowplowanalytics.iglu",
+        "name": "client_error",
+        "format": "jsonschema",
+        "version": "1-0-0"
+    },
+    "type": "object",
+    "required": ["error"],
```

Comment:
does this relate to the `anyOf`?

Reply:
Yes, some `error` is always present. Either `ResolutionError` or `ValidationError`